src/components/Gallery.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useMemo } from 'react';
+import Lightbox from 'react-images';
```

Comment:
good moment to upgrade this dependency as well

Reply:
Looks like it's no longer maintained and the last available [version](https://www.npmjs.com/package/react-images) is in beta.

Reply:
our use case is super simple, let's first see if upgrading it is gonna worth it or not, alternatively we can replace it if there is a similar/lighter lib